lib/init.js

```diff
@@ -91,8 +89,7 @@ var questions = [{
 }, {
   id: 'requirejs',
   question: 'Do you want to use Require.js ?',
-  hint: 'This will add Require.js plugin.\n' +
-    'Press tab to list possible options. Enter to move to the next question.',
+  hint: 'This will add Require.js plugin.\nPress tab to list possible options. Enter to move to the next question.',
```

Comment:
Another alternative
```js
        hint: `This will add Require.js plugin.
Press tab to list possible options. Enter to move to the next question.`,
```

or 

```
hint: 'This will add Require.js plugin.\n'
           + 'Press tab to list possible options. Enter to move to the next question.',
```